zkevm-circuits/src/util/word.rs

```diff
@@ -1,13 +1,14 @@
+#![allow(missing_docs)]
```

Comment:
It's okay to keep this here. This is a good tech debt.
I just want to check if this is a leftover.